libcyphal_demo/src/platform/posix/udp/udp.h

```diff
@@ -0,0 +1,101 @@
+// This software is distributed under the terms of the MIT License.
```

Comment:
Can we somehow reduce code duplication across different demos? We could at least symlink these, although it is certainly not the best solution one could come up with.

Reply:
I'll think about it - maybe at cmake level it would be possible to extract them into a single static library target...